src/i18n/i18ncli.js

```diff
@@ -205,6 +207,17 @@ async function validateFtlFile( ftlPath, options = {} ) {
         keys[item.id.name] = true;
       }
     }
+    if ( item.type === "Message" && item.id.name.startsWith( "date-format" ) ) {
+      item.value.elements.forEach( textElement => {
+        try {
+          format( parseISO( "1970-01-01T00:00:00Z" ), textElement.value, {
+            locale: enUS
+          } );
+        } catch ( error ) {
+          errors.push( `Invalid date/datetime format: ${textElement.value}` );
```

Comment:
Would you make this something more like this?

```javascript
errors.push( `${item.id.name} = ${textElement.value}: ${error.message}` );
```

When this *does* fail, it helps to know exactly what key failed and why.

Reply:
I changed the line. The error does now look more useful:

```
❌ 1 errors found in /.../iNaturalistReactNative/src/i18n/l10n/pt-BR.ftl:
[Error] date-format-month-year = MM AA: Format string contains an unescaped latin alphabet character `A`
```

About the `date-format-short = M/d/aa`: I also think it would be difficult to catch those errors. Hopefully those strings will stabilize within a few iterations :smile:

Reply:
Oh I just noticed I still need to check for items that start with `datetime-format`

Reply:
Now also pushed the checks for `datetime-format`